src/Microsoft.Azure.NotificationHubs.DotNetCore.Tests/NotificationHubClientTest.cs

```diff
@@ -908,6 +908,76 @@ private async Task ScheduleNotificationAsync_SendAppleNativeNotification_GetSucc
             Assert.NotNull(notificationResult.ScheduledNotificationId);
             RecordTestResults();
         }
+
+        [Fact]
+        private async Task GetRegistrationAsync_CreateFcmNativeRegistrationThenGetRegistrationDescription_GetFcmRegistrationType()
```

Comment:
Why are these new tests marked as `private`?

Reply:
@mpodwysocki i have changed to `public`. It is correct accessibility level for new tests?